airflow/__init__.py

```diff
@@ -30,13 +30,8 @@
 # pylint: disable=wrong-import-position
 from typing import Callable, Optional
 
-from airflow import utils
 from airflow import settings
 from airflow import version
-from airflow.utils.log.logging_mixin import LoggingMixin
-from airflow.configuration import conf
-from airflow.exceptions import AirflowException
-from airflow.models.dag import DAG
```

Comment:
I think we also need to remove plugins eventually - but this is a good step (and it will be a bit more complex as we have to make sure plugins are initialized where needed).

Reply:
To do this I'm assuming you'd like to see the `integrate_plugins()` functionality decoupled? 

So for example, `integrate_operator_plugins()` would correspond to the `airflow.operators` module. The function `integrate_sensor_plugins()` would correspond to the `airflow.sensors` module and so on for hooks/macros. 

I'm sure it'd be more complex than what I'm describing, but the idea of only initializing plugins for their respective modules is what you'd be going for, correct?

Reply:
Something like that. I already had a PR for that (but I think I abandon it as there were many changes in the meantime #6696 )

Reply:
But no matter what - if we do it, it should be separate PR.